modules/getting-started-web-console-examining-pod.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-web-console.adoc
+
+:_content-type: PROCEDURE
+[id="getting-started-web-console-examining-pod_{context}"]
+= Examining the Pod
+
+The *Overview* panel enables you to access many features of the `parksmap` deployment. The *Details* and *Resources* tabs enable you to scale application pods, check build status, services, and routes.
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console.
+* You are in the xref:../web_console/odc-about-developer-perspective.adoc#odc-about-developer-perspective[*Developer* perspective].
+* You have a deployed image.
+
+.Procedure
+
+* Click on the `D parksmap` in the *Topology* view to open the *Overview* panel.
++
+.Parksmap deployment
+image::getting-started-examine-pod.png[]
++
+The *Overview* panel includes tabs for *Details*, *Resources*, and *Observe*. The *Details* tab may be displayed by default.
+
++
+.Overview panel tab definitions
+[cols="1,1"]
```

Comment:
Suggest increasing the right column to `,2` or `,3` to give the description more width than the tab name.